refresh SNMP traps page (#32680)
DataDog/documentation: content/en/network_monitoring/devices/snmp_traps.md

- [`ddev`][4] (`pip3 install "datadog-checks-dev[cli]"`)
- [`pysmi`][5] (`pip3 install pysmi`)

Place all your MIBs in a dedicated folder and run:

> ```

Reviewer: ```suggestion
```shell
```

this'll format the command so that it looks like a shell command (prepended with `$`) and also add a Copy button on mouseover (the `$` doesn't get copied)

ddev meta snmp generate-traps-db -o ./output_dir/ /path/to/my/mib1 /path/to/my/mib2 /path/to/my/mib3 ...
```

> The `ddev` tool automatically fetches MIB dependencies online when available. If you encounter missing dependency errors and have the MIB files locally, use the `--mib-sources <DIR>` parameter to specify their location. Ensure filenames match the MIB name (for example, `SNMPv2-SMI`, not `snmp_v2_smi.txt`).

Reviewer: ```suggestion
The `ddev` tool automatically fetches available MIB dependencies. If you encounter missing dependency errors and have the MIB files locally, use the `--mib-sources <DIR>` parameter to specify the location of your local MIB files. Ensure filenames match the MIB name (for example, `SNMPv2-SMI`, not `snmp_v2_smi.txt`).
```
